### ISSUE TITLE
Spring20 app launcher not compatible with Salesforce robotframework library

### ISSUE THREAD

**Issue by edmondo1984** [SFDO-Tooling/CumulusCI#1721]
The following snippet when executed against a scratch org 2020 fails

```
*** Keywords ***

Open Sales
      Open App Launcher
      Select App Launcher App  Sales
```

with the following error:

> Element with locator 'sf:app_launcher.view_all' not found

**Comment by davisagli** [comment]
I'll repeat my answer from the Trailblazer community in case someone else finds it helpful:

I think the problem with your snippet is that the "Select App Launcher App" keyword takes care of opening the app launcher by clicking the waffle button. Since you had already opened the app launcher using "Open App Launcher", that click actually closed the app launcher which is why the "View All" link couldn't be found.

In other words I think it will work if you remove "Open App Launcher".
